src/AutoRest.CSharp/Common/Decorator/EliminateClientDefaultValueTransformer.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoRest.CSharp.Input;
+
+namespace AutoRest.CSharp.Common.Decorator
+{
+    internal class EliminateClientDefaultValueTransformer
```

Comment:
Since this only applies to model properties, shall we rename this to `ModelPropertyClientDefaultValueTransformer`

Reply:
Also we might need to add some comments on this class to explain how and why we are doing this, and/or paste a link to the decision we have made

Reply:
updated. Thanks